scripts/update-contributions.js

```diff
@@ -44,4 +46,7 @@ Group.findAll({
       .catch(err => {
         console.log(`WARNING: ${err.message}`);
```

Comment:
First looking at the more general case of `then`, just use `tap` to clarify return value (it's ignored), and results in more readable syntax so it does make it more maintainable for future devs IMHO:

```
  .tap(bla => console.log(`bla is ${bla}`)
```

Knowing that, I'd simply follow the same pattern for `catch` for consistency and whatever the return value is can be ignored, like [here](https://github.com/OpenCollective/opencollective-api/blob/c0a45245c32e25b0a703fb86a46ecc3a6d7e8706/server/lib/userlib.js#L47-L47).